src/test/java/com/facebook/openwifirrm/modules/ApiServerTest.java

```diff
@@ -559,4 +559,25 @@ void test_runRRM() throws Exception {
 		assertEquals(400, Unirest.put(url + "?mode=test123&venue=" + zone).asString().getStatus());
 		assertEquals(400, Unirest.put(url + "?venue=asdf&algorithm=" + algorithms.get(0)).asString().getStatus());
 	}
-}
+
+
+	@Test
+	@Order(2004)
+	void test_setSystem() throws Exception {
+		String url = endpoint("/api/v1/setSystem");
+
+		// Create JSONObject
+		JSONObject json_obj = new JSONObject();
+		json_obj.put("aa", 'a');
+		json_obj.put("bb", 'b');
+		
+		HttpResponse<String> resp = Unirest
+			.post(url)
+			.body(gson.toJson(json_obj))
```

Comment:
See comments in `ApiServer` above - you should be checking for a valid "command" field in order to return success. Add tests for missing/invalid command (in addition to invalid JSON) in the failure cases.